server/src/main/xsd/gateway-config-201616.xsd

```diff
@@ -1109,6 +1109,66 @@
         </complexContent>
     </complexType>
 
+    <complexType name="turn.restServiceType">
+        <annotation>
+            <documentation>This service provides REST authentication to a TURN server.</documentation>
+        </annotation>
+
+        <complexContent>
+            <restriction base="gateway:ServiceType">
+                <sequence>
+                    <element maxOccurs="1" minOccurs="1"
+                             name="name"
+                             type="gateway:CollapsedString"/>
+
+                    <element maxOccurs="1" minOccurs="0"
+                             name="description"
+                             type="gateway:CollapsedString"/>
+
+                    <element maxOccurs="unbounded" minOccurs="1"
+                             name="accept"
+                             type="anyURI"/>
+
+                    <!-- connect is not supported for echo -->
+
+                    <element maxOccurs="unbounded" minOccurs="0"
+                             name="balance"
+                             type="anyURI"/>
+
+                    <element maxOccurs="unbounded" minOccurs="0"
```

Comment:
nvm, I guess it is legacy implementation.  See https://github.com/kaazing/roadmap/issues/927